File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/Files/IFileData.cs
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Files
{
    public interface IFileData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IFileSize, IFileMIMEType, IFilePathType, IFilePath
    {
        IDataItemList<IDataItemData> Dataset { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Files/IFileData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Files;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IFileData extends IFilePath,IFilePathType,IFileMIMEType,IFileSize,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
